charts/atlantis/Chart.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 appVersion: v0.23.1
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 4.12.0
+version: 5.0.0
```

Comment:
What do you folks think? @runatlantis/maintainers 

Are we ready to do a 5.x? Or should we wait for more interface refactoring before a major upgrade?

Reply:
I'm ok with that since this is a breaking change

Reply:
should we move forward with this?

Reply:
@jamengual We can, just need to coordinate version bumps with pending PRs